R/crosswalk.R

```diff
@@ -21,154 +24,201 @@
 #' crosswalk(r, "GitHub")
 #' }
 #'
-crosswalk <- function(x,
-                      from,
-                      to = "codemeta",
-                      codemeta_context = getOption("codemeta_context",
```

Comment:
I'm undecided about this.  In general, I think it's nicer to use the standard mechanism of providing default function arguments, rather than the more opaque `NULL` that gets handled in the text.  It certainly does make the code cleaner though.  

Probably okay as long as we are explicitly document that it can be set with `options("codemeta_context")` and defaults to `https://doi.org/10.5063/schema/codemeta-2.0`.

Reply:
Maybe a good compromise:
```
codemeta_context = default_context()
```
in the argument list with
```
default_context <- function() {
  getOption("codemeta_context", url_codemeta_schema())
}
```
Or maybe we do not really need `url_codemeta_schema()` and could write direcly
```
default_context <- function() {
  getOption("codemeta_context", "https://doi.org/10.5063/schema/codemeta-2.0")
}
```
And anyway, the behaviour needs to be documented...

Reply:
I documented the default and how to set it in the Roxygen comment for the argument.